src/main/scala/mockws/MockWS.scala

```diff
@@ -59,5 +61,19 @@ object MockWS {
     * @param routes       simulation of the external web resource
     * @param materializer user-defined materializer
     */
-  def apply(routes: Routes, materializer: ActorMaterializer) = new MockWS(routes, () ⇒ Unit)(materializer)
+  def apply(routes: Routes, materializer: ActorMaterializer)(implicit fn: RouteNotDefined) = {
```

Comment:
I'd prefer an `apply` without any implicit parameters:
```
def apply(routes: Routes, materializer: ActorMaterializer, notFoundBehaviour: RouteNotDefined) =
  new MockWS(routes, () ⇒ Unit)(materializer, notFoundBehaviour)
```

Reply:
If we do  that , then in this case it will be picked from implicit scope

Reply:
actually, by thinking again about it, you can let this code as it is now.